UnityProject/Assets/Scripts/Objects/NukeInteract.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Collections;
+using PlayGroup;
+using PlayGroups.Input;
+using UI;
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEngine.Video;
+
+public class NukeInteract : InputTrigger
+{
+    public float cooldownTimer = 2f;
+    public string interactionMessage;
+    public string deniedMessage;
+    [SyncVar]
+    private int nukeCode;
```

Comment:
Probably don't want to be sharing this with every client. Just keep nukeCode on the server and send it when it is needed using a NetworkMessage

Reply:
Yeah, SyncVar means "this info is sent to everyone", definitely not suitable here